website/docs/components/alert/partials/specifications/anatomy.md

```diff
@@ -0,0 +1,25 @@
+## Anatomy
+
+### Page and inline alerts
+
+![Anatomy of the page alert](/assets/components/alert/alert-anatomy-inline.png)
```

Comment:
[small nit] Should the text in the dismiss button label be on one line?

Reply:
Good catch. I'll fix it.